.github/actions/setup_environment/action.yml

```diff
@@ -29,7 +29,7 @@ runs:
   using: "composite"
   steps:
     - name: Set up Python
-      uses: actions/setup-python@v4
+      uses: actions/setup-python@v5.0.0
```

Comment:
Here and in other places, why you are so specific as `5.0.0`? Why not just rely on majors like `5`?

Reply:
AFAIK you specify the literal tag name here and if there isn't a `v5` tag name it fails.